### PR TITLE
Pre-release update for v5.1.1

### PR DESCRIPTION
**Pull request by ValClarkson**
**Checklist:**

 
 - [x ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ x] Have you updated or added documentation for the change, as applicable?
 - [ x] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**

 
 - [ ] New feature
 - [ ] Bug fix
 - [ ] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the current behavior (link to any open issues here)?**

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Other Information**:
[sc -14408]

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,7 @@ PGO_IMAGE_PREFIX ?= crunchydata
 PGO_IMAGE_TAG ?= $(PGO_BASEOS)-$(PGO_VERSION)
 PGO_VERSION ?= $(shell git describe --tags)
 PGO_PG_VERSION ?= 13
-PGO_PG_FULLVERSION ?= 13.4
+PGO_PG_FULLVERSION ?= 13.7
```
Comment (andrewlecuyer) [inline Makefile:9]: Can we update this to the latest PG 14 version instead?

**Review by andrewlecuyer — CHANGES_REQUESTED** [review]
Looks great, thank you!  Just one comment to bump the PG version in the Makefile.

File: Makefile (hunk 1 above)
Comment (ValClarkson) [reply Makefile:9]: will do

Comment (ValClarkson) [reply Makefile]: Updated

**Review by andrewlecuyer — APPROVED** [review]

File: docs/config.toml [hunk 2]
```diff
@@ -26,29 +26,31 @@ disableNavChevron = false # set true to hide next/prev chevron, default is false
...
+operatorVersion = "5.1.1"
+operatorVersionLatestRel5_0 = "5.0.6"
+imageCrunchyPostgres   = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.3-0"
+imageCrunchyPostgresPrivate       = "registry.crunchydata.com/crunchydata/crunchy-postgres:ubi8-14.3-0"
+imageCrunchyPGBackrest = "registry.developers.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi8-2.38-1"
+imageCrunchyPGBackrestPrivate     = "registry.crunchydata.com/crunchydata/crunchy-pgbackrest:ubi8-2.38-1"
+imageCrunchyPGBouncer  = "registry.developers.crunchydata.com/crunchydata/crunchy-pgbouncer:ubi8-1.16-3"
+imageCrunchyExporter   = "registry.developers.crunchydata.com/crunchydata/crunchy-postgres-exporter:ubi8-5.1.1-0"
+imageCrunchyPGAdmin    = "registry.developers.crunchydata.com/crunchydata/crunchy-pgadmin4:ubi8-4.30-1"
 operatorRepository     = "registry.developers.crunchydata.com/crunchydata/postgres-operator"
```
Comment (cbandy) [inline docs/config.toml:38]: 📝 I added a line after `imageCrunchyPGAdmin` here that's causing a conflict.

https://github.com/CrunchyData/postgres-operator/blob/b9e8c53e711e479062cbb558d8af5f750f19efe8/docs/config.toml#L37-L39

Comment (ValClarkson) [reply docs/config.toml:38]: oh i missed the -1 on that one thanks i'll update it now

Comment (ValClarkson) [reply docs/config.toml:38]: @cbandy I looked in the tagging sheet and for pgadmin v5.1.1 release pgadmin is ubi8-4.30-1

**Pull request merged**